embrace-android-sdk/src/integrationTest/kotlin/io/embrace/android/embracesdk/testcases/SpanTest.kt

```diff
@@ -0,0 +1,70 @@
+package io.embrace.android.embracesdk.testcases
+
+import android.os.Build
+import androidx.test.ext.junit.runners.AndroidJUnit4
+import io.embrace.android.embracesdk.IntegrationTestRule
+import io.embrace.android.embracesdk.internal.ApkToolsConfig
+import io.opentelemetry.sdk.common.CompletableResultCode
+import io.opentelemetry.sdk.trace.data.SpanData
+import io.opentelemetry.sdk.trace.export.SpanExporter
+import org.junit.Assert.assertTrue
+import org.junit.Before
+import org.junit.Rule
+import org.junit.Test
+import org.junit.runner.RunWith
+import org.robolectric.annotation.Config
+
+@Config(sdk = [Build.VERSION_CODES.TIRAMISU])
+@RunWith(AndroidJUnit4::class)
+internal class SpanTest {
+    @Rule
+    @JvmField
+    val testRule: IntegrationTestRule = IntegrationTestRule(
+        harnessSupplier = {
+            IntegrationTestRule.newHarness(startImmediately = false)
+        }
+    )
+
+    @Before
+    fun before() {
+        ApkToolsConfig.IS_SDK_DISABLED = false
```

Comment:
You can remove this, I think. This should only be set by tests that need to modify it, and this shouldn't